docs/generated/sparse.COO.rst

```diff
@@ -68,6 +68,7 @@ COO
       COO.to_scipy_sparse
       COO.tocsc
       COO.tocsr
+      COO.__array__
```

Comment:
Do we actually want to document double-underscore functions in the API docs? I'm not sure what the convention is for the Python community. I know documenting them in code is good for potential contributors.

Reply:
[Numpy does document it](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.__array__.html)

And I think in case of `__array__` we should, because it is likely to show unexpected behaviour that users should be able to look up.